lifecycle_msgs/msg/Transition.msg

```diff
@@ -1,25 +1,49 @@
-# Default values for transitions
-# as described on
+# Default values for transitions as described on
```

Comment:
nit: punctuation

Reply:
Addressed in https://github.com/ros2/rcl_interfaces/pull/91